sse2neon.h

```diff
@@ -1469,11 +1478,33 @@ FORCE_INLINE __m128 _mm_cvtpi8_ps(__m64 a)
 // will generate 0x7FFF, rather than 0x8000, for input values between 0x7FFF and
 // 0x7FFFFFFF.
 //
+//   FOR j := 0 to 3
+//     i := 16*j
+//     k := 32*j
+//     IF a[k+31:k] >= FP32(0x7FFF) && a[k+31:k] <= FP32(0x7FFFFFFF)
+//       dst[i+15:i] := 0x7FFF
+//     ELSE
+//       dst[i+15:i] := Convert_FP32_To_Int16(a[k+31:k])
+//     FI
+//   ENDFOR
+//
 // https://software.intel.com/sites/landingpage/IntrinsicsGuide/#text=_mm_cvtps_pi16
 FORCE_INLINE __m64 _mm_cvtps_pi16(__m128 a)
 {
-    return vreinterpret_m64_s16(
-        vmovn_s32(vreinterpretq_s32_m128i(_mm_cvtps_epi32(a))));
+    __m128 i16Min = _mm_set_ps1(INT16_MIN);
+    __m128 i16Max = _mm_set_ps1(INT16_MAX);
```

Comment:
Both `i16Max` and `i32Max` are constant, right?

Reply:
Yes, I'll add the `const` keyword.

Reply:
Done.